app/scripts/components/modelcard.js

```diff
@@ -3,17 +3,43 @@ var exports = (function () {
 
   var ModelCard = Vue.component("model-card", {
     template: "#template-model-card",
+
+    data: function() {
+      return {
+        selected: false
+      };
+    },
+
     props: {
       model: {
         required: true
+      },
+      selectable: {
+        required: false
       }
     },
+
     methods: {
-      toggleActive: function(model) {
-        // we can only toggle on the parent, because we have to clear other actives
-        this.$parent.toggleActive(model);
+      toggleActive: function() {
+        this.model.active = !this.model.active;
+        this.$dispatch("deactivateall", this.model);
```

Comment:
Agreed it would be better to do this with an eventHub as suggested by the documentation. Probably best to asses the whole code-base and update as a separate task and branch.